src/FffGcodeWriter.cpp

```diff
@@ -608,7 +610,6 @@ LayerPlan& FffGcodeWriter::processLayer(const SliceDataStorage& storage, int lay
     }
     int64_t comb_offset_from_outlines = max_inner_wall_width * 2;
 
-    assert(extruder_order_per_layer_negative_layers.size() + layer_nr >= 0 && "Layer numbers shouldn't get more negative than there are raft/filler layers");
```

Comment:
Because `extruder_order_per_layer_negative_layers.size()` and `layer_nr` are both unsigned, so this assertion is caught by the compiler.